en/companion_computer/holybro_pixhawk_rpi_cm4_baseboard.md

```diff
@@ -0,0 +1,216 @@
+# Holybro Pixhawk RPi CM4 Baseboard
+
+# Overview
+
+
+<img src="../../assets/companion_computer/holybro_pixhawk_rpi_cm4_baseboard/HB_Pixhawk_CM4_Basebooard_Main_Photo.jpg" width="580px" title="holybro_pixhawk_rpi_cm4_baseboard" />
+
+The Holybro Pixhawk RPi CM4 Baseboard combine the Pixhawk FC module with the Raspberry Pi CM4 companion computer in on compact form factor with all the connections you need for development. \
+\
+It follows the Pixhawk Connector and Autopilot Bus Standard, allow easy swap of FC Module with any FC that follows the Pixhawk Bus Standard. The FC Module is internally connected to RPi CM4 through TELEM2, and can also be connected via ethernet with a external cable provided.  This baseboard is compatible with both Pixhawk 5X & 6X, and most other flight controller that follow the Pixhawk Autopilot Bus Standard.\
+\
+Recommend minimum specification for RPi CM4:
```

Comment:
@vincentpoont2 The board does not come with an RPi - on your purchase page it says that there is an SKU that has this SKU31107 - but that is not linked anywhere.

Reply:
Ignore that - it is visble when you click the link